libs/platform/auth/src/oauth/components/oauth-handler/oauth-handler.styles.ts

```diff
@@ -0,0 +1,48 @@
+import { css } from 'lit';
+
+export const styles = css`
+  :host {
+    display: flex;
+    flex-direction: column;
+    background-color: var(--oryx-color-primary-300);
+    justify-content: center;
+    height: 100vh;
+    color: var(--oryx-color-canvas-100);
+    --oryx-icon-size: 32px;
+  }
+
+  :host:before {
```

Comment:
What is the aim of this?

Reply:
The handler component does not cover the entire page, and `html` and `body` tag css does not work inside the web component. So I had to use a pseudo element to color the rest of the page.

Reply:
Yeah, looks like a hack, but ok